packages/opentelemetry-basic-tracer/src/types.ts

```diff
@@ -48,6 +48,9 @@ export interface BasicTracerConfig {
    */
   logger?: Logger;
 
+  /** level of logger - 0:disable, 1: error, 2: warn, 3: info, 4: debug  */
```

Comment:
Can we specify that the default is `debug` if not specified? That allows the `console` type to implement the interface in case people want to use it as a convenience. Or does that cause other problems?